packages/lesswrong/themes/userThemes/darkMode.ts

```diff
@@ -89,16 +89,16 @@ const safeColorFallbacks = (palette: ThemePalette) => `
 }
 `;
 
-const colorReplacements = {
+const colorReplacements: Partial<Record<string,string>> = {
   "rgba(255,255,255,.5)": "rgba(0,0,0.5)",
   "hsl(0, 0%, 90%)":    "hsl(0, 0%, 10%)",
   "#F2F2F2":            invertHexColor("#f2f2f2"),
   "rgb(255, 247, 222)": "rgb(50,30,0)",
   "rgb(255, 255, 255)": invertHexColor("#ffffff"),
   "hsl(0,0%,100%)":     invertHexColor("#ffffff"),
   "#FFEEBB":            invertHexColor("#ffeebb"),
-  "rgb(255, 238, 187)": invertColor([255/255.0,238/255.0,187/255.0,1]),
-  "rgb(230, 230, 230)": invertColor([230/255.0,230/255.0,230/255.0,1]),
+  "rgb(255, 238, 187)": colorToString(invertColor([255/255.0,238/255.0,187/255.0,1])),
+  "rgb(230, 230, 230)": colorToString(invertColor([230/255.0,230/255.0,230/255.0,1])),
 };
```

Comment:
```suggestion
} as const;
```
This works even better.